artifacts/operator-configs/controller-conf.yaml

```diff
@@ -58,6 +59,8 @@ data:
   p1-apim-deployment-resources-requests-memory: "2Gi"
   p1-apim-deployment-resources-limits-cpu: "2000m"
   p1-apim-deployment-resources-limits-memory: "2Gi"
+  p1-apim-deployment-env-jvm-heap-memory-xms: "1024m"
+  p1-apim-deployment-env-jvm-heap-memory-xmx: "1024m"
```

Comment:
Lets add this as a global value for all the patterns. Simply we can drop p1 from this variable.